mimic.py

```diff
@@ -45,18 +45,39 @@
 
 import random
 import sys
+from collections import defaultdict
 
 
 def mimic_dict(filename):
-    """Returns mimic dict mapping each word to list of words which follow it."""
-    # +++your code here+++
-    raise NotImplementedError("Get to Work!")
+    f = open(filename, 'r')
```

Comment:
'with open' is now the more 'pythonic' way to open files. It will automatically close files for you